python/cuml/fil/fil.pyx

```diff
@@ -726,7 +721,10 @@ class ForestInference(Base):
             A Forest Inference model which can be used to perform
             inferencing on the random forest model.
         """
-        return self._impl.load_using_treelite_handle(model_handle,
-                                                     output_class,
-                                                     algo, threshold,
-                                                     str(storage_type))
+        self._impl.load_using_treelite_handle(model_handle,
+                                              output_class,
+                                              algo, threshold,
+                                              str(storage_type))
+
+        # DO NOT RETURN self._impl here!!
```

Comment:
I don't think its worth mentioning since I haven't seen it used very much and the finer points could be quite complicated. I put this comment in there because `load_using_treelite_handle` was returning the `_impl` object from a python function which is bad practice.